src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -1573,6 +1573,32 @@ int32_t NaClSysChdir(struct NaClAppThread *natp,
   return retval;
 }
 
+int32_t NaClSysChmod(struct NaClAppThread *natp,
+                     uint32_t             pathname,
+                     int                  mode) {
+  struct NaClApp *nap = natp->nap;
+  char           path[NACL_CONFIG_PATH_MAX];
+  int32_t        retval = -NACL_ABI_EINVAL;
+
+  NaClLog(2, "Cage %d Entered NaClSysChmod(0x%08"NACL_PRIxPTR", "
+          "%d, %d)\n", nap->cage_id, (uintptr_t) natp, pathname, mode);
+
+  if (!NaClAclBypassChecks) {
+    retval = -NACL_ABI_EACCES;
+    goto cleanup;
+  }
```

Comment:
What is the purpose of this? comment it

Reply:
This is in every file/directory related syscall mkdir/chdir/rmdir/getcwd etc... Just followed that to be consistent. Seems like an unesscesary remnant but don't really want to screw with the whole thing right now and keep things consistent.